extensions/rocksdb-repos/DatabaseContentRepository.h

```diff
@@ -38,7 +40,7 @@ namespace repository {
 class StringAppender : public rocksdb::AssociativeMergeOperator {
  public:
   // Constructor: specify delimiter
-  explicit StringAppender() = default;
+  StringAppender() = default;
```

Comment:
minor, but this comment looks stale; I would remove both lines, as the compiler will generate a default constructor anyway